catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/activity/ProjectUploadActivity.java

```diff
@@ -258,11 +259,13 @@ private void showProgressDialogAndUploadProject() {
 		String description = descriptionInputLayout.getEditText().getText().toString().trim();
 
 		if (!project.getName().equals(name)) {
-			ProjectRenameTask
-					.task(project.getDirectory(), name);
-			ProjectLoadTask
-					.task(project.getDirectory(), this);
-			project = ProjectManager.getInstance().getCurrentProject();
+			try {
+				File renamedDirectory = ProjectRenameTask.task(project.getDirectory(), name);
+				ProjectLoadTask.task(renamedDirectory, this);
+				project = ProjectManager.getInstance().getCurrentProject();
+			} catch (IOException e) {
+				Log.e(TAG, "Creating renamed directory failed!", e);
+			}
```

Comment:
May be a new story and too much for a hotfix:
@DinosaurierRex can you add some info for the user, that the renaming failed? What does the user see, when it didn't work? 
@wslany  Is an info enough or should we give the chance to cancel the upload or retry?

Reply:
I don't really think it is necessary to think of some elaborate workflow here. This is a hotfix after all. We can think of some user interaction handling some other time.